### ISSUE TITLE
DatePicker - No proper state is announced for the current date and selected date in Android device

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#16694]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@v7.155.1
- **Browser and OS versions**: **Android Talkback**

### Describe the issue:

When users navigate to the present day, visually it is displayed using the background color but the same information is not conveyed to users who are using **Android Talkback**.

I notice this was fixed by #14806 but it doesn't work for Android devices. In iOS, Voiceover can distinguish current day and selected day and convey corresponding info.
But in **Android**, it still doesn't say anything for the current date as well as selected date.

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar

#### Actual behavior:

No proper info conveyed for current day and selected day

#### Expected behavior:

Talkback can tell users if the button is selected or current day.

**Comment by gouttierre** [comment]
@yanbxice - Thank you for the feedback.

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue?
